From fa7e407ac3598194c3e3c2c0aa88eb97444b0e90 Mon Sep 17 00:00:00 2001
From: Jerin Jacob <jerinj@marvell.com>
Date: Tue, 2 Jul 2019 20:36:19 +0530
Subject: [PATCH 030/154] mk: fix -march for octeontx2 target

Correct the wrong -march=-mcpu=armv8.2-a+crc+crypto+lse for
octeontx2 target. Since rte_cc_has_argument drops invalid
CFLAG and -mcpu=octeontx2 picks up the correct optimization,
this typo is not noticed in performance testing.

Fixes: 01d184798731 ("config: add octeontx2 machine")
Cc: stable@dpdk.org

Signed-off-by: Jerin Jacob <jerinj@marvell.com>
---
 mk/machine/octeontx2/rte.vars.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/machine/octeontx2/rte.vars.mk b/mk/machine/octeontx2/rte.vars.mk
index cbec7f14d19f..e209cf4920c8 100644
--- a/mk/machine/octeontx2/rte.vars.mk
+++ b/mk/machine/octeontx2/rte.vars.mk
@@ -30,5 +30,5 @@
 
 include $(RTE_SDK)/mk/rte.helper.mk
 
-MACHINE_CFLAGS += $(call rte_cc_has_argument, -march=-mcpu=armv8.2-a+crc+crypto+lse)
+MACHINE_CFLAGS += $(call rte_cc_has_argument, -march=armv8.2-a+crc+crypto+lse)
 MACHINE_CFLAGS += $(call rte_cc_has_argument, -mcpu=octeontx2)
-- 
2.14.4

